src/parsers/aws.test.js

```diff
@@ -391,6 +450,110 @@ describe('aws parser', () => {
     ]);
   });
 
+  [
+    // Don't skip a non-empty receive message
+    {
+      envVar: undefined,
+      parsedReqBody: { Action: 'ReceiveMessage' },
+      messageId: 'messageId',
+      expectedShouldSkipSpan: false,
+    },
+
+    // Skip an empty receive message
+    {
+      envVar: undefined,
+      parsedReqBody: { Action: 'ReceiveMessage' },
+      messageId: null,
+      expectedShouldSkipSpan: true,
+    },
+    {
+      envVar: undefined,
+      parsedReqBody: { Action: 'ReceiveMessage' },
+      messageId: undefined,
+      expectedShouldSkipSpan: true,
+    },
+
+    // Don't skip other actions, no matter if they have a message or not
+    {
+      envVar: undefined,
+      parsedReqBody: { Action: 'OtherAction' },
+      messageId: 'messageId',
+      expectedShouldSkipSpan: false,
+    },
+    {
+      envVar: undefined,
+      parsedReqBody: { Action: 'OtherAction' },
+      messageId: null,
+      expectedShouldSkipSpan: false,
+    },
+
+    // Don't skip if parsedReqBody not in expected format
+    { envVar: undefined, parsedReqBody: null, messageId: null, expectedShouldSkipSpan: false },
+    { envVar: undefined, parsedReqBody: undefined, messageId: null, expectedShouldSkipSpan: false },
+    { envVar: undefined, parsedReqBody: {}, messageId: null, expectedShouldSkipSpan: false },
+
+    // Don't skip if env var is explicitly set to false
```

Comment:
please use an array of case groups with descriptions and cases, and then wrap the test in a describe that uses the descriptions so that we know which test group / which test in the group is running

Reply:
Done